File: Backend/UAM-LIFT/src/main/java/ni/edu/uam/UAM_LIFT/enums/EstadoViaje.java
package ni.edu.uam.UAM_LIFT.enums;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum EstadoViaje {
    @JsonProperty("PROPUESTO")
    PROPUESTO,

    @JsonProperty("PROGRAMADO")
    PROGRAMADO,

    @JsonProperty("EN_CURSO")
    EN_CURSO,

    @JsonProperty("FINALIZADO")
    FINALIZADO,

    @JsonProperty("CANCELADO")
    CANCELADO
}

File: Uam Lift/app/src/main/java/ni/edu/uam/uamlift/data/enums/EstadoViaje.kt
package ni.edu.uam.uamlift.data.enums

import com.google.gson.annotations.SerializedName

enum class EstadoViaje {
    @SerializedName("PROPUESTO") PROPUESTO,
    @SerializedName("PROGRAMADO") PROGRAMADO,
    @SerializedName("EN_CURSO") EN_CURSO,
    @SerializedName("FINALIZADO") FINALIZADO,
    @SerializedName("CANCELADO") CANCELADO
}
